### ISSUE TITLE
turning <3 into ❤️ when composing a messsage/post 

### ISSUE THREAD

**Issue by monicaellerose** [bluesky-social/social-app#1688]
The ability when posting text base emojis to auto convert them. in posts/texts

Typing :D or <3 and they don't convert to emojis automatically.  the desired outcome would be 😄  or ❤️ 

This would be a huge quality of life update for using the web composer - not a high priority feature, but nice to have.

**Comment by Sininini** [comment]
It absolutely has to be optional if implemented. 

Also it would be nice one day to be able to type :emotename: so you don't even need to use emote menu like on Discord. That's very low priority imo, but still.

**Comment by chilblane** [comment]
I agree with @Sininini - a lot of users expect to type text emoticons with the intent on keeping them as text, myself included.

**Comment by VickiLanger** [comment]
Also agreed with @Sininini. Being able to type `:meow-facepalm:` in slack or discord is phenomenal. Though, it would be really neat to have similar of bsky, I'm sure this is the lowest of priorities.

For anyone coming across this suggestion, know that there are ways to do this. If you are on [iOS, this should help.](https://support.apple.com/guide/iphone/use-text-replacements-iph6d01d862/ios) For [android users, I think this will get you the same.](https://www.wikihow.com/Add-Custom-Text-Shortcuts-to-Android)
